ep0099/doc.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
```

Comment:
Please update

Reply:
I based this implementation in a simple example I found in SN3218.
updating to 2021.